ruby_review_week_four.rb

```diff
@@ -1,22 +1,47 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
-
+# Variable is a item with something assigned to it.
 # 2. What Ruby method can you use to make a loop from an array?
-
+# Each method
 # 3. In Ruby, what are booleans?
-
+# Boolean is a true or false value. Usually used to evaluate a condition.
 # 4. How do classes relate to objects?
-
+# Classes are used to store objects inside what is called instances.
 # 5. What is an instance variable?
-
+# is variable that lives in a class. They can be used to run class methods.
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
-# 7. What is handy about using gems in Ruby?
+class EveryDayObjects
+  def initialize(shape,color)
+    @shape = shape
+    @color = color
+  end
+
+  def describe
+    puts "The item is #{@shape} and #{@color}"
+  end
+end
 
+cup = EveryDayObjects.new("round","clear")
+cup.describe
+
+# 7. What is handy about using gems in Ruby?
+# Gives access to a custom library to manipulate data or information in ruby. 
 # 8. Define a class called fish, and leave it empty.
 
+class Fish
```

Comment:
again just be precise in the way in which you follow instructions

```rb
class Fish 
end 
```

and then build up the class from there